controller/deis/settings.py

```diff
@@ -136,6 +140,8 @@
     'django.contrib.sites',
     'django.contrib.staticfiles',
     # Third-party apps
+    'django_auth_ldap',
+    'django_fsm',
```

Comment:
Haha, I'm talking about the django_auth_ldap into this list, probably came in some merge I did, had not noticed, I obviously do not use this module, sorry I will remove. :smile: